app/models/taxon_change.rb

```diff
@@ -180,8 +190,11 @@ def commit
       raise RankLevelError, "Output taxon rank level not coarser than rank level of an input taxon's active children"
       return
     end
-    input_taxa.each {|t| t.update_attributes!(is_active: false, skip_only_inactive_children_if_inactive: (move_children? || !active_children_conflict?) )}
+    unless is_a?( TaxonSplit ) && is_branching?
+      input_taxa.each {|t| t.update_attributes!(is_active: false, skip_only_inactive_children_if_inactive: (move_children? || !active_children_conflict?) )}
+    end
     output_taxa.each do |t|
+      next if is_a?( TaxonSplit ) && t.id == taxon_id
```

Comment:
Is it possible to make a branching TaxonSplit with an inactive input? If so, we *do* want to activate it when committed, right?